antenna-documentation/src/site/markdown/processors/ort-downloader.md

```diff
@@ -0,0 +1,12 @@
+## ORT Downloader 
+The ORT Downloader processor downloads the sources of each artifact if no `ArtifactSourceFile` fact is present.
```

Comment:
Just for my understanding: I assume that *if* `ArtifactSourceFile` is present, it points to a local existing file, correct? I.e., it does *not* point to a remote file location where the source code is hosted. Othwiser the sentence would not make sense to me, as I would expect the ORT Download to download remote files.

Reply:
Correct `ArtifactSourceFile` points to a local existing file.d